app/Http/Controllers/SubjectController.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\SubjectRequest;
+use App\Models\Course;
+use App\Models\CourseUser;
+use App\Models\Role;
+use App\Models\Subject;
+use App\Models\SubjectUser;
+use App\Models\Task;
+use Illuminate\Http\Request;
+
+class SubjectController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $courses = Course::with('subjects')->get();
+
+        foreach ($courses->subjects as $subject) {
+            $subject = Subject::load('subjectUsers');
+        }
+        $totalSubjects = count(Subject::all());
+
+        return view('supervisor.manage-subject.list-subjects',
+            compact('courses', 'totalSubjects'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $data['courses'] = Course::where('status', config('number.course.active'))
+            ->get();
+
+        return view('supervisor.manage-subject.create-subject', $data);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Request $request)
+    {
+        $user = auth()->user();
+        $id = $request->subject;
```

Comment:
```suggestion
        $id = $request->subject_id;
```
đặt lại tên biến nhé